src/Symfony/Bridge/Doctrine/Tests/PropertyInfo/DoctrineExtractorTest.php

```diff
@@ -70,6 +70,27 @@ public function testExtract($property, array $type = null)
         $this->assertEquals($type, $this->extractor->getTypes('Symfony\Bridge\Doctrine\Tests\PropertyInfo\Fixtures\DoctrineDummy', $property, array()));
     }
 
+    public function testExtractEmbeddable()
+    {
+        if (!class_exists('Doctrine\ORM\Mapping\Embedded')) {
+            return;
+        }
+
+        $expectedTypes = [new Type(
```

Comment:
thank you, changed PHP version to 5.3 in PhpStorm for this project and fixed declaration